teorija.tex

```diff
@@ -4321,6 +4320,13 @@ \subsection{Hornerjev algoritem}
   $a$ & $a_n$ & $a\krat a_n + a_{n-1}$ & $\dots$ & $\dots$ & \multicolumn{1}{|c}{$p(a)$}
 \end{tabular}
 
+Hornerjev algoritem se uporablja za:
+\begin{itemize*}
+  \item deljenje polinoma $p(x)$ z linearnim polinomom $(x-a)$
+  \item računanje vrednosti polinoma $p(x)$ v točki $a$
+  \item iskanje ničel polinoma $p(x)$
```

Comment:
Ja to imaš prav ampak si v zgornji vrstici naredil isto(vrstica 4304: `Hornerjev algoritem je \textbf{postopek} za \textbf{deljenje} polinoma $p(x)$ z`) zato sem od zgoraj prepisal.